shopping_mall/backend/scripts/migrate_json_to_faq_v2.py

```
@@ -0,0 +1,558 @@
+# ── ChromaDB 동기화 ───────────────────────────────────────────────────────────
+
+def sync_to_chroma(docs: list[FaqDoc]) -> int:
+    """활성 문서를 ChromaDB faq 컬렉션에 일괄 upsert."""
+    synced = 0
+    for doc in docs:
+        if not doc.is_active:
+            continue
+        try:
+            FaqSync.upsert(doc)
+            synced += 1
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)